sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/RntbdTransportClient.java

```diff
@@ -320,7 +320,7 @@ public long id() {
                             RntbdObjectMapper.toJson(throwable));
                     }
                 });
-        }).subscriberContext(reactorContext);
+        }).contextWrite(reactorContext);
```

Comment:
Revert all changes like this as `subscriberContext` was only deprecated.

Reply:
Write follow-up issues for APIs that we deprecated on how to handle when they are removed.